### DISCUSSION TITLE
Migrate from everit-org/json-schema to erosb/json-sKema?

### DISCUSSION THREAD

**Discussion by erosb** [SoftInstigate/restheart#528] [Ideas]
Hello People, I can see in the pom.xml that Restheart still uses the everit-org/json-schema library. That library is pretty much in maintenance mode and doesn't implement newer versions of the json schema spec. Do you plan to migrate to its successor, https://github.com/erosb/json-sKema ?

**Comment by mkjsix** [comment]
Thank you @erosb we are looking into it. We need to check for any API difference and test RESTHeart with the new library.

**Comment by ujibang** [comment]
Hello @erosb 

after a long time we are on it! with the issu https://github.com/SoftInstigate/restheart/issues/751

as long as we understand the old specs are only supported by so we are thinking to use a dual engine approach ( everit-org/json-schema until draft-07 and 2020-12 → json-sKema

do you suggest us this approach? or you think we should only support 2020-12

I guess there must be migration tools out there so maybe being backward compatible is not that important?
